StortSpelProjekt/Vendor/Resources/Models/Cube/cube.obj

```diff
@@ -0,0 +1,19 @@
+v -1.000000 -1.000000 1.000000
+v -1.000000 -1.000000 -1.000000
+v 1.000000 -1.000000 -1.000000
+v 1.000000 -1.000000 1.000000
+v -1.000000 1.000000 1.000000
+v -1.000000 1.000000 -1.000000
+v 1.000000 1.000000 -1.000000
+v 1.000000 1.000000 1.000000
+vt 0.000000 0.000000
+vt 1.000000 0.000000
+vt 1.000000 1.000000
+vt 0.000000 1.000000
+s off
+f 5/1 6/2 2/3 1/4
+f 6/1 7/2 3/3 2/4
+f 7/1 8/2 4/3 3/4
+f 8/1 5/2 1/3 4/4
+f 1/1 2/2 3/3 4/4
+f 8/1 7/2 6/3 5/4
```

Comment:
If this is a "standard cube", can you put this in some folder like: Resources/EngineStandardAssets/cube.obj